web-api/openapi/schemas/document.yml

```diff
@@ -24,3 +24,11 @@ DocumentPropertyId:
   description: Id of a property of a document.
   $ref: './id.yml#/Id'
   example: "title"
+
+DocumentTag:
+  description: A tag of a document can be any non-empty string that consists of alphanumerical characters, common punctuation and whitespace.
+  type: string
+  pattern: '^[a-zA-Z0-9.:,;\-_+*!?%&/\\() ]{1,100}$'
```

Comment:
I don't think we should set a pattern for this.

If we do we must allow unicode letters (e.g. äöüß, Frensh, and tags might even be in japanese even if the articles are not).

This is especially important given that some of our (potential) customers use author names as tags.

So at least all character classes and related non-visible unicode needs to be supported (combinators, zero-sized whitespace, text order changers etc.). For especially rare cases symbols need to be supported, too. And naturally non ascii numeric, too.

I think the easiest would be to allow everything except control characters (assuming bidi control doesn't count as control) as well as tab and newline, but then there is no real reason not to allow them as we still have a max bytes limit anyway.

The regex would be something like `^\P{Control}{1,100}$` (`\P{}` is inverse of `\p` and it's here enough as `\t`, `\r` and `\n` are all control characters. The bidi marks are not covered by `\p{Control}` so this aspect works, too.

Note that  this is 100 characters, while the current patters is 100 bytes (as bytes happen to be characters).

Reply:
Also one problem with that approach is that it might not work for some of the generated sdks.

For example in the browser `/^\P{Control}{1,100}$/` does compile but not behave at all like expected, instead you need to use `/^\P{Control}{1,100}$/u` and no idea if the code gen does so.

So probably better to not set a patter and only set a byte limit :thinking: 

Or only reject `\0`.

Reply:
i guess only rejecting `\0` and having a byte limit is the most flexible, i'll change it.